kolibri/plugins/coach/assets/src/views/common/notifications/ActivityList.vue

```diff
@@ -154,9 +154,6 @@
           const params = {
             ...this.filterParam,
           };
-          if (this.notifications.length) {
-            params.before = this.notifications.slice(-1)[0].id;
```

Comment:
Why has this been removed?

Reply:
The `before` param is already set inside the `moreNotificationsForClass` action https://github.com/AlexVelezLl/kolibri/blob/cdb559e453c0c6468ef95d4c97ad7bfdb342d9ec/kolibri/plugins/coach/assets/src/modules/coachNotifications/index.js#L104. So it was repeated code and thought it was better to keep the one in the `moreNotificationsForClass` action,